claim_from_delivery/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# © 2020 Vanmoof BV (<https://www.vanmoof.com>)
+# © 2004-2009 Tiny SPRL (<http://tiny.be>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Claim on Deliveries",
+    "version": "12.0.1.0.0",
+    "author": "Vanmoof BV, OpenERP SA",
```

Comment:
In order of appearance ;-)

OpenERP SA,Vanmoof BV,Odoo Community Association (OCA)